firmware/mcal/stm32f767/periph/can.h

```diff
@@ -21,8 +21,9 @@ class CanBase : public shared::periph::CanBase {
     CanBase(CAN_HandleTypeDef* hcan) : hcan_(hcan){};
 
     void Setup() {
-        HAL_CAN_ActivateNotification(hcan_, kCanRxActiveInterruptFifo0);
+        ConfigFilters();
         HAL_CAN_Start(hcan_);
+        HAL_CAN_ActivateNotification(hcan_, kCanRxActiveInterruptFifo0);
```

Comment:
are there any implications to adding this line after `HAL_CAN_Start()`? Even if not, i think it should be moved above to be consistent with how it was before.

Reply:
Did this based on some old code that had working rx interrupts. But I do believe it works either way. I'll test it out and will make the change if functional.

Reply:
Looks like it works 👍